### PR TITLE
Portfolio application project 1

### PR DESCRIPTION
**Pull request by i-max-xi**
**Header And Headline Section**

- Nav tag added to menu icon
- Logo section repositioned
- Social media section repositioned

All linters error have been fixed as well

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
### Hi @i-max-xi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### You have submitted a project, which PR has been closed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: index.html [hunk 1]
```diff
@@ -20,25 +29,48 @@
...
-      <h1 class="intro">I'm Max <br> Glad to see you</h1>
-      
+      <img src="Assets/Ellipse8.png" class="small_bal" alt="small_ball_1" />
+      <img src="Assets/Ellipse14.png" class="small_ball" alt="small_ball_2" />
+      <img src="Assets/Ellipse5.png" class="ball" alt="ball" />
+      <img src="Assets/Rectangle7.png" class="bar" alt="bar" />
+      <img src="Assets/Ellipse11.png" class="deep_left" alt="left_ellipse" />
+      <img src="Assets/Rectangle4.png" class="light_right" alt="right_box" />
+      <img src="Assets/Ellipse3.png" class="deep_right" alt="right_ellispe" />
+      <img src="Assets/Rectangle6.png" class="light_left" alt="left_box" />
```
Comment (mmsesay) [inline index.html:39]: - [ ] [OPTIONAL] Please consider placing these images as background images in your CSS.

File: index.html [hunk 2]
```diff
@@ -20,25 +29,48 @@
...
+          <li>
+            <a href="#"><span class="fa fa-hand-peace"></span></a>
+          </li>
+        </ul>
       </div>
     </section>
+    <img src="./Assets/Ellipse4.png" class="roll_ball" alt="roll_ball" />
+    <img src="./Assets/Rectangle5.png" class="roll_bar" alt="roll_bar" />
+    <img src="./Assets/Ellipse12.png" class="bowl" alt="bowl" />
+    <img src="./Assets/Ellipse13.png" class="hook" alt="hook" />
```
Comment (mmsesay) [inline index.html:74]: - [ ] [OPTIONAL] Please consider placing these images as background images in your CSS.

**Review by mmsesay — APPROVED** [review]
Hi @i-max-xi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**